src/box/box.cc

```diff
@@ -4839,6 +5005,7 @@ local_recovery(const struct vclock *checkpoint_vclock)
 	wal_stream_create(&wal_stream);
 	auto stream_guard = make_scoped_guard([&]{
 		wal_stream_abort(&wal_stream);
+		wal_stream_destroy(&wal_stream);
```

Comment:
another `wal_stream_destroy` is needed where the guard is disabled (`stream_guard.is_active = false` line).

Reply:
Done.